cluster/examples/composition/gcp-gkecluster/README.md

```diff
@@ -0,0 +1,40 @@
+# gcp-gkecluster
+
+This example demonstrates Crossplane's nascent support for [composite
+infrastructure resources]. It defines two new kind of resource, both in the
+`gke.example.org` API group:
+
+* `ServicedNetwork` - A GCP VPC network with service networking enabled,
+  allowing instances in this network to privately access GCP services such as
+  CloudSQL.
+* `Cluster` - A GKE cluster with two node pools.
+
+Both resources are defined but not published. Defined resources are cluster
+scoped and intended for use only by infrastructure operators. Any defined
+resource may subsequently be published as available for application operators to
+request - see the neighboring gcp-sqlserver for an example of a published
+resource.
+
+To try this example, first [create a GCP `Provider`] named `example`, then:
+
+1. Define and publish the new resources by running `kubectl apply -f
+   definitions/`.
+1. Create a `ServicedNetwork` by running `kubectl apply -f
+   servicednetwork.yaml`.
+1. Create a `Cluster` by running `kubectl apply -f cluster.yaml`.
+
+Use `kubectl describe` to view the status of your newly created resources:
+
+```shell
+# Make sure the new resources were correctly defined.
+$ kubectl describe infrastructuredefinitions
+
+# Make sure the ServicedNetwork is working:
+$ kubectl describe servicednetworks
+
+# Make sure the Cluster is working
+$ kubectl describe clusters
+```
+
+[composite infrastructure resources]: https://github.com/crossplane/crossplane/pull/1163
+[create a GCP `Provider`]: https://crossplane.io/docs/v0.9/cloud-providers/gcp/gcp-provider.html
```

Comment:
This does not use the latest documentation link here, which should be fine since this page happens to be the same in both. Since this is an example do you intend for this to be updated on release branches?

Reply:
I pushed a commit that avoids linking to a particular doc revision or page - I don't want us to have to bump this and I don't want it to get stale either.